### ISSUE TITLE
Featured avatars should have alt-text description field

### ISSUE THREAD

**Issue by emclaren** [mozilla/hubs#2809]
**Is your feature request related to a problem? Please describe.**
We received feedback from a researcher who did accessibility testing on Hubs with a person using a screen reader that picking an avatar was challenging because they didn't know what any of the avatars were.

Perhaps there should be an alt text field that gets completed by a Hubs team member when an avatar is promoted to "featured" that gives a brief description to make it easier for folks to choose. 

Note - i don't think this should be something that users themselves can add to all avatars, because I worry the alt text would be another thing to check during content review for CPG violations.

**Comment by emclaren** [comment]
Note: this is the feedback from the researcher: 

"The names of the avatar are text, which is screen reader accessible. The names are not very descriptive. It is unclear how the avatars look, if they are gendered or have any other special features"
